### ISSUE TITLE
Bug: Indent rule inconsistent with Prettier ESLint

### ISSUE THREAD

**Issue by justingolden21** [eslint/eslint#15857]
### Environment

Node version: v16.13.0
npm version: v8.1.0
Local ESLint version: v7.32.0 (Currently used)
Global ESLint version: Not found
Operating System: win32 10.0.19044

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```json
{
	"env": {
		"browser": true,
		"es2021": true,
		"node": true
	},
	"extends": ["airbnb-base", "prettier"],
	"parserOptions": {
		"ecmaVersion": 2019,
		"sourceType": "module"
	},
	"plugins": ["prettier", "svelte3"],
	"overrides": [
		{
			"files": ["*.svelte"],
			"processor": "svelte3/svelte3"
		}
	],
	"rules": {
		"prettier/prettier": "error",
		"indent": ["warn", "tab"],
		"quotes": ["warn", "single", { "avoidEscape": true }],
		"no-unused-vars": "warn",
		"no-console": "off",
		"no-tabs": "off",
		"no-restricted-syntax": "off",
		"func-names": "off",
		"quote-props": ["error", "consistent-as-needed"],
		"guard-for-in": "off"
	}
}

```
</details>

```js
export const getColor1 = (obj) =>
	obj.lightness === '-1'
		? 'none'
		: TailwindColors[
				get(settings)[obj.palette === 'accent' ? 'accentColorPalette' : 'baseColorPalette']
		  ][obj.lightness];
```

### What did you expect to happen?

ESLint to be consistent with EsLint Prettier

### What actually happened?

ESLint was inconsistent; see screenshots below

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

![image](https://user-images.githubusercontent.com/30274440/167319730-8d3898aa-8593-404d-8c6e-8bac105d49a2.png)

![image](https://user-images.githubusercontent.com/30274440/167319732-326d9c1d-4873-4bcf-86a0-7526c4415d77.png)

![image](https://user-images.githubusercontent.com/30274440/167319735-1f64282d-20c4-4d71-a910-47ee862a6f0a.png)

**Comment by justingolden21** [comment]
Since I began working on this project again a few days ago, I've run into many of these types of problems. ESLint now complains about quotes and other things and is inconsistent with ESLint Prettier, I get red lines for non-default exports, for ESLint not being able to find imports (in svelte kit) and ESLint thinking that syntax is incorrect such as optional chaining and nullish coalescing which it had no problems with before. Previously, my project was 100% clean, and now I have hundreds of red lines for dozens of new problems... I did find this specific bug interesting, and happily reported it so we can get a rare update to the indentation logic (which I'm sure is very complex) but also if someone would help me determine why my entire project is suddenly red on the same PC with the same packages and same versions and same project and same branch, it would be greatly appreciated. I've wasted five hours on it today alone...

**Comment by justingolden21** [comment]
Another (unrelated) inconsistency:

![image](https://user-images.githubusercontent.com/30274440/167319934-b358bdef-dc0e-4baf-885e-7b1f6568fd54.png)

![image](https://user-images.githubusercontent.com/30274440/167319941-6bd4d19b-173a-4c2c-ac94-5bd2640ba829.png)

**Comment by aladdin-add** [comment]
If you are using prettier, you'll need to disable all stylish rules.
